app/modules/sightings/resources.py

```diff
@@ -57,21 +80,182 @@ def get(self, args):
             'action': AccessOperation.WRITE,
         },
     )
-    @api.login_required(oauth_scopes=['sightings:write'])
+    # @api.login_required(oauth_scopes=['sightings:write'])
     @api.parameters(parameters.CreateSightingParameters())
-    @api.response(schemas.DetailedSightingSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def post(self, args):
         """
         Create a new instance of Sighting.
         """
+
+        data = {}
```

Comment:
I'm trying to get my head around this function and I think it would be simpler if "data" was called request_in or something, 'data' is really quite ambiguous

Reply:
this is just my doing cut-paste of [boilerplate](https://github.com/WildMeOrg/houston/blob/master/app/modules/passthroughs/resources.py#L145-L149) code that is in a [bunch](https://github.com/WildMeOrg/houston/blob/develop/app/modules/encounters/resources.py#L68-L73) of places.  

not really a good excuse tho --  i can start a new trend for this bit of code when i go thru and make some other edits here, yup.

Reply:
updated.